app/views/mailing_list/steps/completed.html.erb

```diff
@@ -19,8 +19,7 @@
     <div class="mailing-list__actions">
       <%= link_to("Book a callback", callbacks_steps_path, class: "button") %>
       <div>
-        <small>or call us now:</small>
-        <p class="telephone">0800 389 2500</p>
+        <p class="telephone">or call us now on 0800 389 2500</p>
```

Comment:
@philspray This isn't actually a link (it is not in prod either) so is not clickable on mobile - so I'm not sure the change is worth it as is (and actually it makes the formatting a bit funny)

Should we change it to be a link? Or just keep the original with the two lines?

![image](https://user-images.githubusercontent.com/87642394/214383179-0f4d29b7-e648-4231-b982-405da51cd6e3.png)